apps/librelingo-server/go.mod

```diff
@@ -0,0 +1,5 @@
+module librelingo-server/m
```

Comment:
I would maybe call it librelingo-standalone-server

Reply:
The whole "app" or just the go module (i.e.: should the subdirectory in /apps be changed as well?)

Reply:
I would change the subdirectory as well, I think it will make it more clear.

I'd imagine people would be looking for the word "standalone" and also there might be another "librelingo-server" later